h/migrations/versions/530268a1937c_add_password_last_updated_column_to_.py

```diff
@@ -0,0 +1,25 @@
+"""Add password last updated column to accounts model
+
+Revision ID: 530268a1937c
+Revises: 43645baa68b2
+Create Date: 2015-12-14 17:07:44.757878
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '530268a1937c'
+down_revision = '43645baa68b2'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('user', sa.Column('password_updated',
+                                    sa.DateTime(),
+                                    server_default=sa.func.now(),
```

Comment:
As discussed in slack, don't add the `server_default` on creation, as that makes this a rather unfriendly migration (locks table, rewrites every row). So:

1. Add a line to this migration which does `op.alter_column('user', 'password_updated', server_default=sa.func.now())`.
2. Add an intermediate data migration which adds the data to rows with missing values for this column. See [this migration](https://github.com/hypothesis/h/blob/0d4a6a0449ffd7983adfef945afae04f8e37cc11/h/migrations/versions/1f82507200d7_fill_in_missing_pubids.py) for an example.
3. Update the last migration (the one that does `ALTER COLUMN password_updated NOT NULL`) to reference the data migration as its parent.